prboom2/src/m_cheat.c

```diff
@@ -66,9 +66,6 @@
 
 #define plyr (players+consoleplayer)     /* the console player */
 
-//e6y: for speedup
-static int boom_cheat_route[MAX_COMPATIBILITY_LEVEL];
```

Comment:
Boom compat lost at some point?

Reply:
There used to be multiple cheat code interpreters that read input keys in different ways. At some point I merged down to one, and it looks like this variable was used in the old iteration, so it can be removed now.